client/src/pages/_document.js

```diff
@@ -1,13 +1,13 @@
 import { Html, Head, Main, NextScript } from "next/document";
 
 export default function Document() {
-  return (
-    <Html lang="en">
-      <Head />
-      <body>
-        <Main />
-        <NextScript />
-      </body>
-    </Html>
-  );
+	return (
+		<Html lang="en">
+			<Head />
+			<body>
+				<Main />
+				<NextScript />
+			</body>
+		</Html>
+	);
```

Comment:
No idea why these changes show that I'm adding space or made any changes here. It is safe to ignore.